firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/TCA9548A.h

```diff
@@ -8,7 +8,7 @@
 
 #include "I2CMux.h"
 #include "Pufferfish/Driver/Testable.h"
-#include "Pufferfish/HAL/HAL.h"
+#include "Pufferfish/HAL/STM32/HAL.h"
```

Comment:
Let's only include `Pufferfish/HAL/Interfaces/DigitalOutput.h` - drivers should not be including or using anything from `HAL/STM32`, so this PR is a great opportunity to fix this problem in the old code here.